packages/cli/test/utils/simulation/interfaces.ts

```diff
@@ -104,8 +108,7 @@ export interface CLNode {
   readonly url: string;
   readonly api: Api;
   readonly keyManager: KeyManagerApi;
-  readonly localKeys: SecretKey[];
-  readonly remoteKeys: SecretKey[];
+  readonly keys: CLClientKeys;
```

Comment:
but there can be both local and remote kets in the CL

Reply:
Yes there can be, but as per @dapplion we don't want to make SIM tests complex and mix up the keys. You can skill have local or remote keys in multiple CL nodes.

Reply:
CLClientKeys represents both as an enum